src/main/java/RoutineManagerModel/Workout.java

```diff
@@ -0,0 +1,31 @@
+package RoutineManagerModel;
+
+import java.util.ArrayList;
+
+public class Workout {
+
+    ArrayList<String> workout;
+    public Workout() {
+
+        this.workout = new ArrayList<String>();
+    }
+
+    public void AddExercise(Exercise exercise, int index) {
+
+        //this.workout.add(index, exercise.);
```

Comment:
What exactly is the purpose of this method? It seems the line of code that adds a workout is commented out.